web/portal/brand/src/entities/Wholesale/Form.tsx

```diff
@@ -18,10 +21,14 @@ const Form = (props: EntityFormProps): JSX.Element => {
     match,
   });
 
+  const aboutMe = useStoreState((state) => state.clientSession.aboutMe.profile);
+  const hasInvoicesFeature = aboutMe?.features.includes('invoices') || false;
+  const hasBillingFeature = aboutMe?.features.includes('billing') || false;
+
   const groups: Array<FieldsetGroups | false> = [
     {
       legend: _('Basic Configuration'),
-      fields: ['name', 'billingMethod'],
+      fields: ['name', hasBillingFeature && 'billingMethod'],
```

Comment:
I'm ok with this format, as long as linter check passes, but I would have make it multiline when using conditions, seems more clear to me:

```
fields: [
  'name',
  hasBillingFeature && 'billingMethod'
],

Reply:
it was multiline before the linter changed it :sweat_smile:

Reply:
Doh!